bitnami/metrics-server/values.yaml

```diff
@@ -17,6 +17,10 @@ image:
   ##
   pullPolicy: Always
 
+  # For supporting pulling from private registries
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
```

Comment:
For other solutions we are using something like:
```yaml
  ## Optionally specify an array of imagePullSecrets.
  ## Secrets must be manually created in the namespace.
  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
  ##
  # pullSecrets:
  #   - myRegistrKeySecretName
```

We've the option commented and we added some documentation